tests/Shared/WorkloadTesting/BuildEnvironment.cs

```diff
@@ -127,6 +127,10 @@ public BuildEnvironment(bool useSystemDotNet = false, TestTargetFramework target
         // Set DEBUG_SESSION_PORT='' to avoid the app from the tests connecting
         // to the IDE
         EnvVars["DEBUG_SESSION_PORT"] = "";
+        // Avoid using the msbuild terminal logger, so the output can be read
+        // in the tests
+        EnvVars["VsTestUseMSBuildOutput"] = "false";
+        EnvVars["MSBUILDENSURESTDOUTFORTASKPROCESSES"] = "1";
```

Comment:
@baronfel @rainersigwald - should this just happen automatically in CI?

Reply:
dotnet test should be doing this for you if you use an invocation of it that requires this flag: https://github.com/dotnet/sdk/blob/176e8ca72f5b9111649c49ca43a71d1dd9904bd0/src/Cli/dotnet/commands/dotnet-test/Program.cs#L61

In general MSBuild should not allow worker nodes to stomp all over the central nodes' stdout, VSTest is doing a bad here.

Reply:
@nohwnd can you advise here?

Reply:
Since rc1 (IIRC) the dotnet test output is tied to the terminal logger status. When Terminal Logger is off the new output is also off. `MSBUILDTLENABLED=0` or `-tl:false` should be enough to disable the new output.